src/cluster/managers/documentation/CommandDocumentationManager.ts

```diff
@@ -88,6 +88,31 @@ export class CommandDocumentationManager extends DocumentationTreeManager {
         };
     }
 
+    #listCommandNames(names: readonly string[]): string {
+        const charLimit = discord.getLimit('embed.field.value') - 6;
+        if (names.reduce((p, c) => p + c.length + 2, 0) - 2 <= charLimit)
```

Comment:
why you adding 2 in the reduce then removing 2 from the total?

Reply:
Thats what happens to the length when you .join and array, the filler gets added between each element, so it gets added `.length - 1` times
`['a','b','c'].join(', ') === 'a, b, c'`
`('a'.length + 2) + ('b'.length + 2) + ('c'.length + 2) - 2`